docs/tutorials/getting_started.md

```diff
@@ -0,0 +1,135 @@
+# Getting started
+
+## Prerequisites
+
+1. Python installed on your system.
+2. Access to a microgrid system supported by the `frequenz.sdk` or you can use
+   the sandbox.
+3. Basic knowledge of microgrid concepts.
+4. Familiarity with
+   [Channels](https://frequenz-floss.github.io/frequenz-channels-python/latest/).
+
+## Set up your Frequenz development environment
+
+### Setting up a Python environment
+
+Before you begin, make sure you have a working Python environment. You can
+create a virtual environment to manage dependencies.
+
+```bash
+# Create a virtual environment (optional but recommended)
+python -m venv frequenz_sdk_env
+source frequenz_sdk_env/bin/activate
+# On Windows, use 'frequenz_sdk_env\Scripts\activate'
+```
+
+### Install the Frequenz SDK
+
+Install the Frequenz SDK. For this tutorial, this is all you need, but in the
+future, you might also need additional packages based on your project
+requirements.
+
+```bash
+pip install frequenz-sdk  # Install the Frequenz SDK
+```
```

Comment:
Not sure if this shouldn't also be a link to [`../#installation`](https://github.com/frequenz-floss/frequenz-sdk-python/blob/v0.x.x/docs/index.md#installation) to avoid repetition and the risk of getting out of sync as things move.

Maybe the whole installation can be merged with the first item of the pre-requirements (having python)?